### ISSUE TITLE
"Short Name" multilingual field for Equipment?

### ISSUE THREAD

**Issue by jdgwf** [jdgwf/battletech-tools#37]
Should we have a "Short Name" multilingual field. I've noticed on the Record Sheet criticals that the AutoCannon Ammo text was exceeding the critical box size. I "quickfixed" this by renaming it to Ammo (AC/XX)... I think this is a hack.

Do you think having a .shortName field would be helpful?

```
.name {
"en-US": "Ammo (Autocannon/20)"
},
.shortName {
"en-US": "Ammo (AC/20)"
}
```

The RecordSheet functions can easily be coded to default to the .name

**Comment by Moonsword22** [comment]
Yeah, I'd noticed some of the ammo names were unwieldy.  I can look into adding that later for English weapons.
